src/main/java/com/napier/sem/DAO.java

```diff
@@ -208,4 +207,27 @@ public static String getWhereCondition(String areaFilter, String areaName)
 
         return executeStatement(statementString, App.CAPITAL_CITY);
     }
+
+    /**
+     * Use cases 6.1-6.3
+     * Constructs an SQL query to fetch the top N populated cities in a specific area, and executes the query.
+     *
+     * @return An ordered list of cities in a defined area sorted by descending population
+     */
+    public ArrayList<Record> topNCapitalCitiesIn(String areaFilter, String areaName, Integer n) {
+        String whereCondition = getWhereCondition(areaFilter, areaName);
+
+
+        // Define the SQL query as a string
+        String statementString = "SELECT city.name, city.population, country.region, country.continent, country.name AS country \n" +
+                "FROM city\n" +
+                "    JOIN country ON city.countrycode = country.code\n" +
+                "WHERE city.id = country.capital \n" +
```

Comment:
this `WHERE` could be part of the join, which might be a little more efficient. i'm happy leaving it like this, or i can should you how i'd change it.